### ISSUE TITLE
Promise does not resolve anymore

### ISSUE THREAD

**Issue by hadrienl** [aurelia/dialog#204]
**I'm submitting a bug report**
- **Library Version:**
  1.0.0-beta.3.0.0

**Please tell us about your environment:**
- **Operating System:**
  OSX 10.11.6
- **Node Version:**
  6.2.0
- **NPM Version:**
  3.8.9
- **JSPM OR Webpack AND Version**
  AU
- **Browser:**
  Chrome 51
- **Language:**
  ESNext

**Current behavior:**
I've removed dialog css. I have not set a transition/animation on dialog closing/opening.
I open a dialog, I wait for the promise to be resolve, I close the dialog, the promise is never resolved. It is if the dialog have an animation.

**Expected/desired behavior:**
The promise should be resolved when calling controller.ok() or cancel() even if it is not animated.

**Comment by PWKad** [comment]
For now I would just set the animation to be immediate instead of covering this scenario.

**Comment by hadrienl** [comment]
Yes, I put back the animation, but I lost an hour to find why my promise never resolved (I wasn't the one who rewrite the dialog css), so it should be good to fix this or at least to warn about in the doc.

**Comment by PWKad** [comment]
Sure I'll mark it as such.

**Comment by PWKad** [comment]
Enhancement - support transition-less closing of the dialog

**Comment by StrahilKazlachev** [comment]
@PWKad Just transition-less closing or both - transition-less open and close? Because if both the `ignoreTransitions` option should do the job.

**Comment by PWKad** [comment]
True @hadrienl this option should cover your scenario, can you confirm?  If so let's close this out.

**Comment by StrahilKazlachev** [comment]
@hadrienl I don't know if it's documented. Usage: `.open`/`.openAndyYeldController` `({ ignoreTransitions: true })`

**Comment by hadrienl** [comment]
Ok, so I think you should add this information to the documentation and close the issue.

**Comment by PWKad** [comment]
@hadrienl PR welcome here, else I've tagged this issue and we should be able to get to it relatively easily.

**Comment by hadrienl** [comment]
I have found another use case when promise fails to resolve. I've setup a form in my dialog with a css theme which set transition on input's :focus state. Then, if I click with my mouse on submit button in same time I blur my input, the input is transitioning while the dialog is closed and the promise does not resolve. I've fixed this with a simple workaround : removing all transitions in all elements included in a dialog, but I would prefer the bug to be fixed.
